/ktp_data_manager:
  ros__parameters:
    dev_id: "KECDSEMITB001"
    map_id: "220"
    firmware_version: "1.0.5.21"